definitions/pipelines/somatic_exome.cwl

```diff
@@ -126,6 +126,13 @@ inputs:
     custom_clinvar_vcf:
         type: File?
         secondaryFiles: [.tbi]
+    manta_call_regions:
+        type: File?
+        secondaryFiles: [.tbi]
+    manta_non_wgs:
+        type: boolean?
```

Comment:
Should this have a default of `true` in this exome pipeline?

Reply:
Good call, added in the latest commit